wordle-web/src/views/WordEditor.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+    <div class="text-center">
+    <v-pagination v-model="page" :length="totalPages" :total-visible="6" rounded="circle"></v-pagination>
+    <v-row>
+        <v-col v-for="item in paginatedItems" :key="item.wordId" cols="2">
+            <v-list-item @click="openDialog(item)">
+                <v-list-item-title>{{ item.text }}</v-list-item-title>
+            </v-list-item>
+        </v-col>
+    </v-row>
+    <v-pagination v-model="page" :length="totalPages" :total-visible="6" rounded="circle"></v-pagination>
+        <v-row justify="center">
+            <v-col cols="2">
+                <v-select label="Items per page" v-model="perPage" :items="[30, 60, 120, 240]"></v-select>
+            </v-col>
+            <v-col cols="2">
+                <v-text-field v-model="search" label="Search for word"></v-text-field>
+            </v-col>
+            <v-col cols="2">
+                <v-btn color="primary" text @click="addnewword = true">Add new word</v-btn>
+            </v-col>

```

Comment:
can add maxlength=5 to text field to only allow the user to enter 5 chars for input validation (same on search box)
![image](https://github.com/IntelliTect-Samples/EWU-CSCD379-2023-Spring/assets/61523476/e54374c9-3f1e-499e-b9e2-a2d77316e937)